LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift

```diff
@@ -35,61 +34,55 @@ class ViewController: UIViewController {
+       present(nextvc, animated: true)
     }
     
     @objc func signUP(_ sender: UIButton) {
-        print("signUp Button Tapped")
-        let storyBoard = UIStoryboard(name: "PhoneNumberViewController", bundle: nil)
-        
- let nextvc = storyBoard.instantiateViewController(withIdentifier: "PhoneNumberViewController") as! PhoneNumberViewController
+        let storyBoard = UIStoryboard(name:StoryBoardsID.SignUp.rawValue, bundle: nil)
+        let nextvc = storyBoard.instantiateViewController(withIdentifier: StoryBoardsID.SignUp.rawValue) as! SignUpViewController
        present(nextvc, animated: true)
     }
 }
 
-extension ViewController {
+extension LogInSignUpViewController {
  
     func parentStackViewConstraint() {
-        parentStackView.translatesAutoresizingMaskIntoConstraints                                                           = false
-        parentStackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 170).isActive           = true
-        parentStackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 0).isActive     = true
-        parentStackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: 0).isActive   = true
-        parentStackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -10).isActive     = true
+        parentStackView.translatesAutoresizingMaskIntoConstraints = false
+        parentStackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 170).isActive = true
+        parentStackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 0).isActive = true
+        parentStackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: 0).isActive = true
+        parentStackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -10).isActive = true
         //      adding views In StackView.
         addViewsInParentStackView()
-        
     }
     
     func imageStackViewConstraint() {
-        leezeitImageStackView.translatesAutoresizingMaskIntoConstraints                                               = false
-        leezeitImageStackView.centerXAnchor.constraint(equalTo: parentStackView.centerXAnchor).isActive               = true
-        leezeitImageStackView.topAnchor.constraint(equalTo: parentStackView.bottomAnchor, constant: -10).isActive     = true
+        leezeitImageStackView.translatesAutoresizingMaskIntoConstraints = false
+        leezeitImageStackView.centerXAnchor.constraint(equalTo: parentStackView.centerXAnchor).isActive = true
+        leezeitImageStackView.topAnchor.constraint(equalTo: parentStackView.bottomAnchor, constant: -10).isActive = true
         leezeitImageStackView.addArrangedSubview(leezeitImage)
```

Comment:
does it work if we dont put image in stack?

Reply:
i would prefer stack for better constraints